pandas/core/reshape/pivot.py

```diff
@@ -541,7 +541,10 @@ def crosstab(
     rownames = _get_names(index, rownames, prefix="row")
     colnames = _get_names(columns, colnames, prefix="col")
 
-    common_idx = get_objs_combined_axis(index + columns, intersect=True, sort=False)
+    common_idx = None
+    pass_objs = [x for x in index + columns if isinstance(x, (ABCSeries, ABCDataFrame))]
+    if pass_objs:
```

Comment:
i see what you are doing here, i guess ok, this is slightly awkward because get_objs_combined_axis now can't take an empty list.